layers/generate_layers.py

```diff
@@ -65,6 +75,64 @@ def generatePolygonAmenity(args, amenity, filename):
     run(["tippecanoe", f"{filename}.geojson", "-o", f"{filename}.pmtiles"])
 
 
+def makeMRN():
+    # Remove files from any previous run
+    try:
+        os.remove("Major_Road_Network_2018_Open_Roads.zip")
+        os.remove("mrn.pmtiles")
+        shutil.rmtree("mrn")
+    except:
+        pass
+
+    # Get the shapefile
+    run(
+        [
+            "wget",
+            "https://maps.dft.gov.uk/major-road-network-shapefile/Major_Road_Network_2018_Open_Roads.zip",
+        ]
+    )
+    run(["unzip", "Major_Road_Network_2018_Open_Roads.zip", "-d", "mrn"])
+
+    # Convert to GeoJSON, projecting to WGS84
+    run(
+        [
+            "ogr2ogr",
+            "-f",
+            "GeoJSON",
+            "mrn/mrn.geojson",
+            "-t_srs",
+            "EPSG:4326",
+            "mrn/Major_Road_Network_2018_Open_Roads.shp",
+        ]
+    )
+
+    # Clean up the file
+    path = "mrn/mrn.geojson"
+    print(f"Cleaning up {path}")
+    gj = {}
+    with open(path) as f:
+        gj = json.load(f)
+        # Remove unnecessary attributes
+        del gj["name"]
+        del gj["crs"]
+        for feature in gj["features"]:
+            # Remove all properties except for "name1", and rename it
```

Comment:
The MRN shapefile has some other fields, but nothing looks particularly useful for our simple display purposes. Can revisit after getting feedback from users.